### PR TITLE
kaitie/added button started container for rubric pages

### PR DESCRIPTION
**Pull request by kobryan0619**
I am looking for some feedback on the Curriculum Writer Experience start of things before I get too far.   This PR is just a first step in getting the full experience up and running.  This...

**1. Adds a button to the "lesson edit page"**

image

Future work:  Make the button work.  AND, if time, make the button say "edit rubric" or "add rubric" depending if there is an existing rubric or not. 
Feedback needed on: Location of the button - is this what we expect?

**2. Creates the "new rubric" page (http://localhost-studio.code.org:3000/rubrics/new)**
Future work:  Add the content to the page as well as make it functional.

**3. Creates the "edit rubric" page (http://localhost-studio.code.org:3000/rubrics/15/edit)**
Future work: Add content to the page, make it functional, and pass in the correct rubric ID to the URL

## Links

[AITT-59](https://codedotorg.atlassian.net/jira/software/projects/AITT/boards/70?selectedIssue=AITT-59)

## Follow-up work

See above for follow-up work.  Additionally, we need to restrict this URL to level-builder access only. 

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.jsx [hunk 1]
```diff
@@ -145,17 +145,28 @@ class ActivitiesEditor extends Component {
...
+              onMouseDown={this.handleAddActivity}
+              className="btn add-activity"
+              style={styles.addActivity}
+              type="button"
+            >
+              <i style={{marginRight: 7}} className="fa fa-plus-circle" />
+              Activity
+            </button>
+          )}
           <button
```
Comment (bethanyaconnor) [inline apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.jsx]: This should be a link, possibly styled as a button, as it'll take the user to a new page (likely in a new tab). 

Also curious on your thoughts on this button/link linking to the new rubric page immediately? I agree that dynamically linking to new or edit is out of scope though

File: dashboard/config/routes.rb [hunk 2]
```diff
@@ -334,6 +334,8 @@
       end
     end
 
+    resources :rubrics, only: [:edit, :new], param: 'key'
```
Comment (bethanyaconnor) [inline dashboard/config/routes.rb]: I think the param will be the `id` here, which I think is the Rails default, so you should be able to remove the `param:` param

File: apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.jsx (hunk 1 above)
Comment (kobryan0619) [reply apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.jsx]: Thanks - I was able to read up on the button/Button/a-link discussion on the accessibility channel and made this change.

I did make it link to the new rubric page immediately here, too.

**Review by bethanyaconnor — APPROVED** [review]
LGTM!

**Pull request merged**